include/CLI/App.hpp

```diff
@@ -1763,7 +1763,7 @@ class App {
         if(name_.empty()) {
             return std::string("[Option Group: ") + get_group() + "]";
         }
-        if(aliases_.empty() || !with_aliases || aliases_.empty()) {
+        if(aliases_.empty() || !with_aliases) {
```

Comment:
Assuming this was not trying to check something else?

Reply:
I suspect it was copied and ended up duplicated condition